run_iwyu_tests.py

```diff
@@ -167,6 +176,11 @@ def RegisterFilesForTesting(rootdir, pattern):
 
 
 if __name__ == '__main__':
+  unittest_args, additional_args = Partition(sys.argv, '--')
+  if additional_args:
+    iwyu_test_util.SetIwyuPath(additional_args[0])
+  sys.argv = unittest_args
```

Comment:
I think it is cleaner to do `unittest.main(argv=unittest_args)` if I understand [Python docs](https://docs.python.org/2/library/unittest.html#unittest.main) correctly.